docs/localstorage-disks.md

```diff
@@ -0,0 +1,53 @@
+# Local Storage Placement for VM Disks
+
+
+## Introduction
+
+Virtual Machines are able to have a DataVolume disks that are based on Local Storage PVs. Local Storage PVs are bound to a specific node.
+It might happen to have an `Unschedulable` `VMI`, when the PVC is bound to a PV on a different node than virt-launcher pod. 
```

Comment:
this might read a bit better somethign like this...

"Since DataVolumes involve preparing storage with a image before being consumed by the VMI, it's possible to result in an `Unschedulable VMI` in the event that a VMI can not be scheduled to the node the local storage PV was previously pinned to."

Reply:
yes, easier to read